src/components/Subheader/Subheader.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import useStyles from '../Subheader/Subheader.styles.js';
+import CloseIcon from '@material-ui/icons/Close';
+import PropTypes from 'prop-types';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+export default function Subheader(props) {
+  const { variant, mainLabel, subLabel, image, testClick } = props;
+  const isMobile = useMediaQuery('(max-width:1023px)');
+  const classes = useStyles({ variant, isMobile });
+  const [showSubheader, setSubheader] = useState(true);
+  const {
+    subheaderContainer,
+    subheader,
+    subheaderItems,
+    subheaderIcon,
+    mainText,
+    subText,
+    closeIcon,
+  } = classes;
+  const handleClick = () => {
+    setSubheader(!showSubheader);
+  };
+
+  return (
+    <div className={subheaderContainer}>
+      {showSubheader ? (
+        <div className={subheader} data-testid="page-title">
+          <div className={subheaderItems}>
+            <div className={subheaderIcon}>{image}</div>
+
+            <div className={mainText}>{mainLabel}</div>
+            {isMobile ? null : <div className={subText}>{subLabel}</div>}
+          </div>
+          <button
+            onClick={testClick || handleClick}
+            className={closeIcon}
+            title="closeSubheader"
+          >
```

Comment:
use es6 on attributes of 2 or more...

`<button
            {...{onClick: testClick || handleClick, className: closeIcon, title: "closeSubheader"}}
          >`

Reply:
Updated button to es6 syntax in commit 73d9de9db39116691ef5a43f35a6240f22df9b59